### PR TITLE
Logging cleanup

### PR DESCRIPTION
**Pull request by chrisadubois**

### CODE REVIEW

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js [hunk 1]
```diff
@@ -120,9 +119,8 @@ export default class LocusInfo extends ChildEmitter {
    */
   onFullLocus(locus, eventType) {
     if (!locus) {
-      console.log('locus object undefined');
+      LoggerProxy.logger.error('LocusInfo->onFullLocus#Locus object passed as argument was invalid, continuing.');
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js]: Would something like this be considered `verbose`?

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js [hunk 2]
```diff
@@ -303,56 +290,29 @@ export default class LocusInfo extends ChildEmitter {
...
-      console.info('LocusInfo:controls got updated');
     }
   }
 
   updateConversationUrl(conversationUrl) {
     if (conversationUrl && !isEqual(this.conversationUrl, conversationUrl)) {
       this.conversationUrl = conversationUrl;
-      this.emitChanges(EVENTS.CONVERSATION_URL_UPDATE, conversationUrl);
-      console.info('LocusInfo:conversationUrl got updated');
+      this.emitGlobal(EVENTS.CONVERSATION_URL_UPDATE, conversationUrl);
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js]: The scope v. global concept is interesting.

File: packages/node_modules/@webex/plugin-meetings/src/meeting/audio.js [hunk 3]
```diff
@@ -19,7 +19,7 @@ const handleTransition = (audio) => {
 };
 
 const triggerEvent = (meeting, state, result, trigger, type) => {
```
Comment (mattnorris) [inline packages/node_modules/@webex/plugin-meetings/src/meeting/audio.js]: This has reached the argument limit 😉

Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/meeting/audio.js]: lol

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js (hunk 2 above)
Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js]: kudos to @arun3528 !

File: packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js (hunk 1 above)
Comment (chrisadubois) [reply packages/node_modules/@webex/plugin-meetings/src/locus-info/index.js]: anything that is routed through EventProxy will be considered verboseEvents, I don't have a general verbose config yet, just a general verboseEvents :) to keep it isolated to that. if we want to add a verboseErrors config we could do that.

**Comment by chrisadubois** [comment]
https://github.com/arun3528/spark-js-sdk/pull/10

**Pull request closed without merging**